commit 06c21f7423e7eabbe85dd18d4de8e9f6bc69400d
Author: YueHaibing <yuehaibing@huawei.com>
Date:   Fri Jun 1 19:25:48 2018 +0800

    ath10k: fix incorrect size of dma_free_coherent in ath10k_ce_alloc_src_ring_64
    
    [ Upstream commit 5a211627004e2cddd0ab8b9df19e5fb0bbe97634 ]
    
    sizeof(struct ce_desc) should be a copy-paste mistake
    just use sizeof(struct ce_desc_64) to avoid mem leak
    
    Fixes: b7ba83f7c414 ("ath10k: add support for shadow register for WNC3990")
    Signed-off-by: YueHaibing <yuehaibing@huawei.com>
    Signed-off-by: Kalle Valo <kvalo@codeaurora.org>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/net/wireless/ath/ath10k/ce.c b/drivers/net/wireless/ath/ath10k/ce.c
index 3b96a43fbda4..18c709c484e7 100644
--- a/drivers/net/wireless/ath/ath10k/ce.c
+++ b/drivers/net/wireless/ath/ath10k/ce.c
@@ -1512,7 +1512,7 @@ ath10k_ce_alloc_src_ring_64(struct ath10k *ar, unsigned int ce_id,
 		ret = ath10k_ce_alloc_shadow_base(ar, src_ring, nentries);
 		if (ret) {
 			dma_free_coherent(ar->dev,
-					  (nentries * sizeof(struct ce_desc) +
+					  (nentries * sizeof(struct ce_desc_64) +
 					   CE_DESC_RING_ALIGN),
 					  src_ring->base_addr_owner_space_unaligned,
 					  base_addr);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1209_06c21f7423e7eabbe85dd18d4de8e9f6bc69400d.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1209_06c21f7423e7eabbe85dd18d4de8e9f6bc69400d.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
